### ISSUE TITLE
Drop PHP 5.3 support

### ISSUE THREAD

**Issue by Trainmaster** [PHPOffice/PHPWord#1304]
This is:

- [ ] a bug report
- [x] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpword)

# Expected Behavior

PHPWord requires at least PHP 5.6. Maybe it would be good to use the same PHP requirements as in https://github.com/PHPOffice/PhpSpreadsheet/blob/develop/composer.json

What do you think? @troosan 

# Context

PHP 5.3 reached its EOL on 2014-08-14.

**Comment by troosan** [comment]
Indeed, that's on the todo list. But first I want to address all the existing pull requests.
Once that's done I'll drop 5.3 support.

**Comment by dragonattack** [comment]
Some of the dependencies are throwing error at php 5.3.3 so, php 5.3 support is dropped really.

```Parse error: syntax error, unexpected '[' in .../public_html/contract/vendor/zendframework/zend-stdlib/src/StringUtils.php on line 34```

**Comment by troosan** [comment]
@dragonattack It all depends with which PHP version you are running composer.
If you run composer with PHP 5.3.3 it will load version 2.2.10, which should not give you this error

**Comment by rasamassen** [comment]
This issue should be marked closed.

**Comment by Progi1984** [comment]
Done in 1.0.0
